<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoChoose</class>
 <widget class="QWidget" name="UserInfoChoose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>374</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>284</width>
    <height>374</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>284</width>
    <height>374</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MyWidgetRegister" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>276</width>
     <height>364</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>276</width>
     <height>364</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>276</width>
     <height>364</height>
    </size>
   </property>
   <widget class="QLabel" name="labelHead">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>头像</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNickname">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>32</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>昵称</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>134</width>
      <height>60</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>134</horstretch>
      <verstretch>60</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>134</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>134</width>
      <height>60</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>已参加</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelJoinNum">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>134</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>134</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>134</width>
      <height>60</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>已创建</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelCreated">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>281</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>----------------------------------------------</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>276</width>
      <height>221</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>276</width>
       <height>74</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>-6</x>
        <y>4</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">text-align:left;
padding-left:20px;</string>
      </property>
      <property name="text">
       <string>绑定社交账号</string>
      </property>
      <property name="icon">
       <iconset resource="imagelogin.qrc">
        <normaloff>:/Login/C:/Users/ASUS/Pictures/Image/yyy.jpg</normaloff>:/Login/C:/Users/ASUS/Pictures/Image/yyy.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>147</y>
       <width>276</width>
       <height>74</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>-6</x>
        <y>2</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">text-align:left;
padding-left:20px;</string>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
      <property name="icon">
       <iconset resource="imagelogin.qrc">
        <normaloff>:/Login/C:/Users/ASUS/Pictures/Image/yyy.jpg</normaloff>:/Login/C:/Users/ASUS/Pictures/Image/yyy.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>276</width>
       <height>74</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>-6</x>
        <y>-8</y>
        <width>291</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">text-align:left;
padding-left:20px;</string>
      </property>
      <property name="text">
       <string>个人信息设置</string>
      </property>
      <property name="icon">
       <iconset resource="imagelogin.qrc">
        <normaloff>:/Login/C:/Users/ASUS/Pictures/Image/yyy.jpg</normaloff>:/Login/C:/Users/ASUS/Pictures/Image/yyy.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyWidgetRegister</class>
   <extends>QWidget</extends>
   <header location="global">mywidgetregister.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imagelogin.qrc"/>
 </resources>
 <connections/>
</ui>
